ddtrace/contrib/flask/patch.py

```diff
@@ -70,11 +72,21 @@
 )
 
 
-def get_version():
+def _get_version():
     # type: () -> str
     return get_version_for_package("flask")
 
 
+def get_version():
```

Comment:
`get_version` is defined in every integration. If we plan to rename this function in the flask integration we should do it for our other integrations. Here's the PR that introduced this function: https://github.com/DataDog/dd-trace-py/commit/ab6321c9dd2beccecbc8e8028cd4144689db98fe.

Reply:
I think the plan is to go one integration at a time. @rachelyangdog lmk if that sounds right.

Reply:
yes! i was going to make a PR for each integration